quantestpy/circuit.py

```diff
@@ -17,34 +17,31 @@
 
 
 def assert_equal_to_operator(
+        circuit: Union[TestCircuit, str],
```

Comment:
Note that qiskit.QuantumCircuit cannot be added to the type hint unless qiskit is installed.